app/tests/spec/models/auth_brokers/oauth-redirect.js

```diff
@@ -49,14 +95,15 @@ define(function (require, exports, module) {
       sinon.stub(broker, 'finishOAuthFlow').callsFake(() => {
         return Promise.resolve();
       });
+      // sinon.spy(broker, 'finishOAuthFlow');
     });
 
     it('has the `signup` capability by default', () => {
       assert.isTrue(broker.hasCapability('signup'));
     });
 
-    it('does not have the `handleSignedInNotification` capability by default', () => {
-      assert.isFalse(broker.hasCapability('handleSignedInNotification'));
+    it('have the `handleSignedInNotification` capability by default', () => {
+      assert.isTrue(broker.hasCapability('handleSignedInNotification'));
```

Comment:
[We don't want this capability to change](https://github.com/mozilla/fxa-content-server/pull/6960/files#diff-e28aae2c15d692d7632811c383025a76L39) with this update, can you revert this?